pkg/synopsysctl/cmd_update.go

```diff
@@ -587,9 +587,9 @@ var updateBlackDuckAddEnvironCmd = &cobra.Command{
 
 // updateBlackDuckAddRegistryCmd adds an Image Registry to a Black Duck instance
 var updateBlackDuckAddRegistryCmd = &cobra.Command{
-	Use:           "addregistry BLACK_DUCK_NAME REGISTRY",
-	Example:       "synopsysctl update blackduck addregistry <name> docker.io\nsynopsysctl update blackduck addregistry <name> docker.io -n <namespace>",
-	Short:         "Add an Image Registry to a Black Duck instance",
+	Use:           "addregistry BLACK_DUCK_NAME IMAGE",
+	Example:       "synopsysctl update blackduck addregistry <name> gcr.io/blackducksoftware/blackduck-cfssl:2019.6.0\nsynopsysctl update blackduck addregistry <name> gcr.io/blackducksoftware/blackduck-cfssl:2019.6.0 -n <namespace>",
```

Comment:
@mphammer  instead of addRegistry, can we have addImageRegistry or updateImageRegistry. It's so confusing..